### PR TITLE
[docs] update java quickstart

### LINKED ISSUE #8351: [DOC] Instructions for JAVA application development needs more intermediate steps
The stepping details are not listed by the web-page
`https://docs.yugabyte.com/latest/quick-start/build-apps/java/ysql-jdbc`

Please provide an example for Java app by listing the steps for readers to copy and paste.  These steps should be continuing those in "Create local cluster" page, `https://docs.yugabyte.com/latest/quick-start/create-local-cluster/linux`.  The following is an example on what steps should be done.

**1. Create the project "MySample".**
%% _mvn archetype:generate -DgroupId=com.yugabyte -DartifactId=MySample -DarchetypeArtifactId=maven-archetype-quickstart -DinteractiveMode=false_

%% _cd MySample_

**2. Add the dependency and install it.**
Edit file "./pom.xml" to add the following.
```
<dependency>
    <groupId>org.postgresql</groupId>
    <artifactId>postgresql</artifactId>
    <version>42.2.14</version>
</dependency>
```
Install the added dependency
%% _mvn install_

**3. Add a new program.**
Copy the following JDBC contents to a new file name "src/main/java/com/yugabyte/HelloSqlApp.java
```
package com.yugabyte;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.ResultSet;
import java.sql.SQLException;
import java.sql.Statement;

public class HelloSqlApp {
  public static void main(String[] args) throws ClassNotFoundException, SQLException {
    Class.forName("org.postgresql.Driver");
    Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte",
                                                  "yugabyte", "yugabyte");
    Statement stmt = conn.createStatement();
    try {
        System.out.println("Connected to the PostgreSQL server successfully.");
        stmt.execute("DROP TABLE IF EXISTS employee");
        stmt.execute("CREATE TABLE IF NOT EXISTS employee" +
                     "  (id int primary key, name varchar, age int, language text)");
        System.out.println("Created table employee");

        String insertStr = "INSERT INTO employee VALUES (1, 'John', 35, 'Java')";
        stmt.execute(insertStr);
        System.out.println("EXEC: " + insertStr);

        ResultSet rs = stmt.executeQuery("select * from employee");
        while (rs.next()) {
          System.out.println(String.format("Query returned: name = %s, age = %s, language = %s",
                                           rs.getString(2), rs.getString(3), rs.getString(4)));
        }
    } catch (SQLException e) {
      System.err.println(e.getMessage());
    }
  }
}
```

**4. Execute the new program.**
%% _mvn install exec:java -Dexec.mainClass=com.yugabyte.HelloSqlApp_

### PR DESCRIPTION
**Pull request by polarweasel**
Fixes #8351

Build preview is here: https://deploy-preview-8353--infallible-bardeen-164bc9.netlify.app/latest/quick-start/build-apps/java/ysql-jdbc/

### CODE REVIEW

**Comment by polarweasel** [comment]
@nocaway the maven output is really noisy. Can you suggest how to make it quieter? `mvn -q` seems to be insufficient... Ideally the output of the final step should just be the app's output, and nothing from Maven. Other maven operations could definitely be quieter, too. 😄

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md [hunk 1]
```diff
@@ -56,135 +56,112 @@ This tutorial assumes that:
...
+    </project>
+    ```
+
+1. Install the added dependency.
+
+    ```sh
+    $ mvn install
+    ```
+
+1. Make a new directory for your app.
```
Comment (nocaway) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md:109]: You don't this "mkdir" step.
The above step of "mvn install" should have already made the directory "com/yugabyte".

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md [hunk 2]
```diff
@@ -56,135 +56,112 @@ This tutorial assumes that:
...
 
-You should see the following as the output.
+    You should see the following as the output:
 
-```
-Created table employee
-Inserted data: INSERT INTO employee (id, name, age, language) VALUES (1, 'John', 35, 'Java');
-Query returned: name=John, age=35, language: Java
-```
+    ```output
```
Comment (nocaway) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md:163]: Missing the first line in the output.  The posted code should also output the following text.
```
Connected to the PostgreSQL server successfully.
```

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md [hunk 3]
```diff
@@ -56,135 +56,112 @@ This tutorial assumes that:
...
+        -DartifactId=MySample \
+        -DarchetypeArtifactId=maven-archetype-quickstart \
+        -DinteractiveMode=false
+
+    $ cd MySample
+    ```
+
+1. Replace the contents of the `pom.xml` file with the following:
+
+    ```xml
```
Comment (nocaway) [inline docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md:73]: "mvn" might create and require different "pom.xml" headers for different environments.  The doc could be worded such as "After adding the postgres dependency, your pom.xml content would look similar to the following."

Adding postgres dependency:
```
        <dependency>
            <groupId>org.postgresql</groupId>
            <artifactId>postgresql</artifactId>
            <version>42.2.14</version>
        </dependency>
```

Potential content for file pom.xml
```
< The content that you posted>
```

**Review by nocaway — COMMENTED** [review]
Using option `-q` in mvn commands would be great.  Thank you.
My review comments are inline

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md (hunk 3 above)
Comment (polarweasel) [reply docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md:73]: updated

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md]: removed!

File: docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md (hunk 2 above)
Comment (polarweasel) [reply docs/content/latest/quick-start/build-apps/java/ysql-jdbc.md:180]: added back in :)

**Comment by polarweasel** [comment]
@nocaway updated! everything works as I'd expect now. Please take a final review and we can publish.

**Pull request merged**